src/com/rental/utils/SqlWorker.java

```diff
@@ -0,0 +1,159 @@
+            deleteProductStatement = connector.connect.prepareStatement(SQL_DELETE_PRODUCT);
+            Statement statementProduct = connector.connect.createStatement();
+            Statement statementProductName = connector.connect.createStatement();
+            Statement statementUserRole = connector.connect.createStatement();
+            Statement statement = connector.connect.createStatement();
+            products_rs = statementProduct.executeQuery(PRODUCTS_QUERY);
+            products_name_rs = statementProductName.executeQuery(PRODUCTS_NAME_QUERY);
+            users_rs = statement.executeQuery(USERS_QUERY);
+            usersRole = statementUserRole.executeQuery(USERS_QUERY);
+        } catch (SQLException ex) {
+            ex.printStackTrace();
+        }
+    }
+
+    public void createNewProduct() {
+        try {
+            addProductStatement.setString(1, Communicator.enterProductName());
+            addProductStatement.setString(2, Communicator.enterCategory());
+            addProductStatement.setInt(3, Communicator.enterSqlProductCount());
+            addProductStatement.setInt(4, 0);
+            addProductStatement.executeUpdate();
+        } catch (SQLException ex) {
+            ex.printStackTrace();
+        }
+    }
+
+    public void showAllMovie() {
+        try {
+            while (products_rs.next()) {
+                int productId = products_rs.getInt(PRODUCT_ID_FIELD);
+                String productName = products_rs.getString(PRODUCT_NAME_FIELD);
+                String productCategory = products_rs.getString(PRODUCT_CATEGORY_FIELD);
+                String productCount = products_rs.getString(PRODUCT_COUNT_FIELD);
+                String productReviews = products_rs.getString(PRODUCT_REVIEWS_FIELD);
+                System.out.println(productId + " " + productName + " " + productCategory + " " + productCount + " " + productReviews);
+            }
+        } catch (SQLException ex) {
+            ex.printStackTrace();
+        }
+    }
+
+    public void deleteProduct() {
+        try {
+            deleteProductStatement.setInt(1, Communicator.productToDelete());
+            deleteProductStatement.executeUpdate();
+            deleteProductStatement.close();
+        } catch (SQLException ex) {
+            ex.printStackTrace();
+        }
+    }
+
+    private static String nameOdLoggedUser;
+
+    public void login() {
+        boolean temp = true;
+        try {
+            while (temp) {
+                String enterLogin = Communicator.enterLoginField();
+                String enterPassword = Communicator.enterPasswordField();
+                while (users_rs.next()) {
+                    if (users_rs.getString(USER_NAME_FIELD).equals(enterLogin) && users_rs.getString(USER_PASSWORD_FIELD).equals(enterPassword)) {
+                        Communicator.correctDataInfo();
+                        nameOdLoggedUser = enterLogin;
```

Comment:
Referencing to static field from a non-static context is a major synchronization issue! Comments are applicable for all usages in the project, please revise it.